integrations/terraform/tfschema/token/custom_types.go

```diff
@@ -0,0 +1,62 @@
+package token
+
+import (
+	"context"
+	"fmt"
+	apitypes "github.com/gravitational/teleport/api/types"
+	"github.com/gravitational/teleport/integrations/terraform/tfschema"
+	"github.com/hashicorp/terraform-plugin-framework/attr"
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+	"github.com/hashicorp/terraform-plugin-framework/tfsdk"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+)
+
+// GenSchemaLabels returns Terraform schema for Labels type
+func GenSchemaLabels(ctx context.Context) tfsdk.Attribute {
+	return tfschema.GenSchemaLabels(ctx)
+}
+
+// GenSchemaBoolOptionsFixed returns Terraform schema for BoolOption type
+func GenSchemaBoolOptionFixed(_ context.Context) tfsdk.Attribute {
+	return tfsdk.Attribute{
+		Optional: true,
+		Type:     types.BoolType,
+	}
+}
+
+func CopyFromBoolOptionFixed(diags diag.Diagnostics, tf attr.Value, o **apitypes.BoolOption) {
```

Comment:
We probably ought to add a short description here as to what is differentf/fixed?